source/_repos.html.slim

```diff
@@ -14,3 +14,11 @@ ul
     | For bugs with rspec-expectations:&nbsp;
     = link_to 'https://github.com/rspec/rspec-expectations', 'https://github.com/rspec/rspec-expectations'
     |  (matchers)
+  li
+    | For bugs with rspec-support:&nbsp;
+    = link_to 'https://github.com/rspec/rspec-support', 'https://github.com/rspec/rspec-support'
+    |  (string encoding, failed expectation diffing, Ruby feature detection, etc)
+  li
+    | For bugs with rspec-dev:&nbsp;
+    = link_to 'https://github.com/rspec/rspec-dev', 'https://github.com/rspec/rspec-dev'
+    |  (test runners)
```

Comment:
rspec-dev is actually a set of tools for helping develop with RSpec, so it's not test runners really.